lms/static/js/commerce/views/receipt_view.js

```diff
@@ -61,6 +63,12 @@ var edx = edx || {};
                 $courseNamePlaceholder.text(responseData.name);
             });
         },
+        renderUserFullNamePlaceholder: function (username) {
+            var $userFullNamePlaceholder = $(".full_name_placeholder");
```

Comment:
Nitpick: please use ```'``` for strings to maintain stylistic consistency within the file.